src/app/components/hmcts-global-header/hmcts-global-header.component.html

```diff
@@ -57,7 +57,7 @@
       <nav class="hmcts-primary-navigation transparent-background" aria-label="Primary navigation">
         <ul class="hmcts-primary-navigation__list">
           <li class="hmcts-primary-navigation__item" *ngFor="let item of items" >
-            <a class="hmcts-primary-navigation__link" (click)="onEmitSubMenu(item)" [attr.aria-current]="item.active ? true: null" [routerLink]="item.href">{{item.text}}</a>
+            <a class="hmcts-primary-navigation__link" (click)="onEmitSubMenu(item)" [attr.aria-current]="(tab === item.text && !(currentUrl === '/cases/case-search')) ? true: null" [routerLink]="item.href">{{item.text}}</a>
```

Comment:
`item.active` should be working just fine. If it's not, change the mechanism that sets `item.active`, rather than change this binding.

Reply:
Resolving this on the understanding that the change will be made in a subsequent PR.